int8 board
int8 channel
float64 value
---
int8 error